### PR TITLE
Tiny clean up.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
#148

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
@shenli @siddontang  PTAL

File: server/expire_cache.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+)
+
+type cacheItem struct {
+	key    uint64
+	value  interface{}
+	expire time.Time
+}
+
+// ExpireRegionCache is an expired region cache.
+type ExpireRegionCache struct {
```
Comment (siddontang) [inline server/expire_cache.go]: why still export?

**Comment by siddontang** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**